com.github.stsdc.monitor.json

```diff
@@ -0,0 +1,347 @@
+{
+  "app-id": "com.github.stsdc.monitor",
+  "runtime": "io.elementary.Platform",
+  "runtime-version": "7.3",
+  "sdk": "io.elementary.Sdk",
+  "command": "com.github.stsdc.monitor",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
+    "--socket=system-bus",
+    "--device=shm",
+    "--device=all",
+    "--share=network",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
```

Comment:
you should never be setting this.

Reply:
The same as here: https://github.com/flathub/net.nokyan.Resources/blob/master/net.nokyan.Resources.json#L16

Reply:
two wrongs don't make a right.

Reply:
If wrong is not right then why it is accepted?

Reply:
it is being QUESTION. You didn't explain why you need them. Just that you cut and pasted from somewhere.

Reply:
It is needed to detect if a process is an app, to find its desktop file and icon.